lib/keypress.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const readline = require('readline');
+const emitKeypressEvents = require('keypress')
```

Comment:
We already use a modified version of [keypress here](https://github.com/enquirer/enquirer/blob/master/lib/keypress.js), can we do this without adding a dependency?

Reply:
It could be done, but there seems to be some nuance in the keypress library handling cleanup of listeners, avoiding multiple events firing, and possibly handling multiple keys arriving at once via buffers (not exactly sure on that last one).  I wouldn't trust my own implementation to do that.

I think keeping a well used and stable dependency like keypress around is less prone to bugs than a custom implementation in enquirer, and future changes to enquirer will have less surface area to make mistakes against if the readline functionality remains enclosed fully in the 3rd party module.

Alternatively I could copy in the single keypress index file into enquirer ?  Which approach would you prefer ?